arrays.dd

```diff
@@ -218,9 +218,12 @@ $(H2 $(LNAME2 array-copying, Array Copying))
 ---------
 int[3] s;
 int[3] t;
+int[] d = new int[3];
 
-s[] = t;           // the 3 elements of t[3] are copied into s[3]
-s[] = t[];         // the 3 elements of t[3] are copied into s[3]
+d[] = t;           // the 3 elements of t are copied into d (unlike "d = t")
+s[] = t;           // the 3 elements of t are copied into s
+s[] = t[];         // the 3 elements of t are copied into s
+s = t;             // also works (because s is a static array)
```

Comment:
That's true, but not unique to "s = t;" - it's also a requirement of "s[] = t;".
What I'm trying to point out is that "d = t;" would not mean the same thing.